src/args/parse-helm-fetch-args.ts

```diff
@@ -0,0 +1,27 @@
+import args from "https://deno.land/x/args@2.0.7/wrapper.ts"
+import { CollectOption } from "https://deno.land/x/args@2.0.7/flag-types.ts"
+import { Text } from "https://deno.land/x/args@2.0.7/value-types.ts"
+
+const textOption = (flag: string, alias?: readonly string[]) =>
+  CollectOption(flag, {
```

Comment:
CollectOption подразумевает, что может быть передано несколько флагов
```suggestion
  PartialOption(flag, {
```

Reply:
Oh, I'm sorry. Yes